guides/scopes.md

```diff
@@ -48,5 +50,14 @@ Same, if you are not responsible for a scope, you are always encouraged to check
 | [Cloud Scripts](https://github.com/meilisearch/cloud-scripts) | [alallema](https://github.com/alallema/) |
 |||
 | [docs-scraper](https://github.com/meilisearch/docs-scraper)| [bidoubiwa](https://github.com/bidoubiwa/) |
+| [js-project-boilerplate](https://github.com/meilisearch/js-project-boilerplate)| [bidoubiwa](https://github.com/bidoubiwa/) |
 |||
 | [WordPress integration](https://github.com/meilisearch/docs-scraper) (Stopped)| [curquiza](https://github.com/curquiza/) |
+
+### Access
+
+When a member of the integration team owns a repository, he/she has full access to this repository, i.e. Admin access on GitHub.
+
+However, a great power comes with a great responsibility. It means:
```

Comment:
```suggestion
However, a great power comes with great responsibilities. It means:
```

Thats the original sentences and sinces there are multiple bullet points it is more accurate?

Reply:
😇 
https://www.google.com/search?channel=trow5&client=firefox-b-d&q=a+great+power+comes+with+a+great+responsibility

Reply:
Why don't we use the real sentence in full:
`However, with great power comes great responsibility`